routes/articles.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+
+const express = require("express");
+const router = express.Router();
+const articlesDB = require("../db/articles.js");
+
+router.get("/", (req, res) => {
+  res.render("articleIndex", { theArticles: articlesDB.getArticles() });
+});
+
+router.get("/new", (req, res) => {
+  res.render("newArticle");
+});
+
+router.get("/:urlTitle", (req, res) => {
+  let title = req.params.urlTitle;
+  let article = articlesDB.getSingleArticle(title);
+  res.render("articleSingle", article);
+});
+
+router.get("/:urlTitle/edit", (req, res) => {
+  let title = req.params.urlTitle;
+  let article = articlesDB.getSingleArticle(title);
+  res.render("editArticle", article);
+});
+
+router.post("/", (req, res) => {
+  if (
+    req.body.title !== "" &&
+    req.body.description !== "" &&
+    req.body.author !== ""
+  ) {
+    articlesDB.addArticle(
+      req.body.title,
+      req.body.description,
+      req.body.author
+    );
+    res.render("articleIndex", {
+      theArticles: articlesDB.getArticles(),
+      success: "Success!"
+    });
+  } else {
+    res.render("newArticle", {
```

Comment:
This should redirect to your `/new` route and the error should be kept in a variable outside this function.